src/compute/src/logging/initialize.rs

```diff
@@ -10,24 +10,26 @@ use std::collections::BTreeMap;
 use std::rc::Rc;
 use std::time::{Duration, Instant};
 
+use crate::arrangement::manager::TraceBundle;
+use crate::extensions::arrange::{KeyCollection, MzArrange};
+use crate::logging::compute::ComputeEvent;
+use crate::logging::{BatchLogger, EventQueue, SharedLoggingState};
```

Comment:
Nit: Move down to after imports from other crates.

Our style is:
* std imports
* other crate imports
* this crate imports
* Separated by a newline.